test.js

```diff
@@ -13,3 +13,12 @@ test(t => {
 	t.is(m('🐶', 5), '🐶');
 	t.is(m('👍🏿', 1), '👍🏻');
 });
+
+test('Color name constants', t => {
```

Comment:
`Color name constants` => `constants`

And if you're gonna give this a title, you need to name the other one too.